vulnerabilities/package_managers.py

```diff
@@ -29,6 +29,7 @@
 
 from aiohttp import ClientSession
 from aiohttp.client_exceptions import ClientResponseError
+from aiohttp.client_exceptions import ServerDisconnectedError
```

Comment:
Using aiohttp or request or urllib in various places to just fetch does not make sense to me. We should have only one way (one or a few functions) to fetch across the codebase

Reply:
`aiohttp` is used only for making tons of requests to package managers to fetch  versions. 

Other than that I have tried to use `requests` wherever possible.

Reply:
FYI `urllib` is used in msr importer because it is more convenient for downloading and reading csv from memory than `requests` .